packages/component/src/Utils/FocusRedirector.tsx

```diff
@@ -30,8 +35,14 @@ const FocusRedirector: FC<FocusRedirectorProps> = ({ className, onFocus, redirec
   //             When using NVDA in browse mode, press up/down arrow keys will focus on this redirector.
   //             This redirector is designed to capture TAB only and should not react on browse mode.
   //             However, reacting with browse mode is currently okay. Just better to leave it alone.
-
-  return <div className={className} onFocus={handleFocus} tabIndex={0} />;
+  return (
+    <div
+      aria-hidden={ariaHidden}
+      className={className}
+      onFocus={handleFocus}
+      tabIndex={redirectRef?.current ? TAB_INDEX_ENABLED : TAB_INDEX_DISABLED}
```

Comment:
It is ok to use just:
```suggestion
      tabIndex={redirectRef?.current ? 0 : -1}
```

If lint fails, can disable the rule for the file

Reply:
fixed